src/components/PopularCoursesLayout/PopularCoursesContainer/PopularCard/styledPopularCard.ts

```diff
@@ -10,7 +10,7 @@ export const StyledPopularCardHeader = styled.p`
   line-height: 50px;
   @media(max-width: 1270px){
     font-size: 26px;
-    margin-bottom: 20px;
+    /* margin-bottom: 20px; */
   }
```

Comment:
you decided not to add margin here, cause you forgot or cause this design sucks?)))